docs/user/setup.rst

```diff
@@ -7,7 +7,7 @@ Scala Native has the following build dependencies:
 
 * Java 8 or newer
```

Comment:
I am aware that this line comes from the venerable 0.4.0-M2 description but I think it
is problematic.  To make any progress at all, we can not get mired in fixing every
point 10 places to the right of the decimal. However, some are worth a bit of effort
to fix as we find them.  The interest of new devos is a precious and perishable thing.
(as he preaches to the quire.)

I think this should something like (rough) 
``` 
Java 8 or newer for projects using Scala Native
Java 8 only to build Scala Native project itself.
```
One would think the second line belongs in the Contributor's Guide. Indeed it might
be duplicated there. But by the time a new devo gets to the Contributor's Guide
the damage has been done.

Times change so I might be behind the curve. These days I use
.bash file to set a Java 8 environment to build SN itself.  Even then
I continue to hose myself on interactive builds by forgetting to
run the project login script and trying to build with the default
Java 11 on my system.